### PR TITLE
feat: route of smzdm product

### PR DESCRIPTION
**Pull request by chesha1**
## Involved Issue / 该 PR 相关 Issue

Close #

## Example for the Proposed Route(s) / 路由地址示例

```routes
/smzdm/product/zm5vzpe
```

## New RSS Route Checklist / 新 RSS 路由检查表
  
- [x] New Route / 新的路由
  - [x] Follows [Script Standard](https://docs.rsshub.app/joinus/advanced/script-standard) / 跟随 [路由规范](https://docs.rsshub.app/zh/joinus/advanced/script-standard)
- [ ] Anti-bot or rate limit / 反爬/频率限制
  - [ ] If yes, do your code reflect this sign? / 如果有, 是否有对应的措施?
- [x] [Date and time](https://docs.rsshub.app/joinus/advanced/pub-date) / [日期和时间](https://docs.rsshub.app/zh/joinus/advanced/pub-date)
  - [x] Parsed / 可以解析
  - [x] Correct time zone / 时区正确
- [ ] New package added / 添加了新的包
- [ ] `Puppeteer`

### CODE REVIEW

File: lib/routes/smzdm/product.ts [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+        const price = $(this).find('.z-highlight').text();
+        const title = altText + ' ' + price;
+        const description = $(this).find('.feed-block-descripe').text().replaceAll(/\s+/g, '');
+        const item = {
+            title,
+            link,
+            description,
+        };
+        items.push(item);
+    });
```
Comment (TonyRL) [inline lib/routes/smzdm/product.ts:51]: Please use [Array.prototype.map()](https://developer.mozilla.org/docs/Web/JavaScript/Reference/Global_Objects/Array/map) instead

File: lib/routes/smzdm/product.ts [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+                item.pubDate = pubDate;
+
+                if (item.description === '阅读全文') {
+                    item.description = $('p[itemprop="description"]').first().html() as string;
+                }
+
+                return item;
+            }
+        })
+    );
```
Comment (TonyRL) [inline lib/routes/smzdm/product.ts:73]: Please [cache](https://docs.rsshub.app/joinus/advanced/use-cache) the returned result.

File: lib/routes/smzdm/product.ts [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+        requireConfig: false,
+        requirePuppeteer: false,
+        antiCrawler: false,
+        supportBT: false,
+        supportPodcast: false,
+        supportScihub: false,
+    },
+    radar: [
+        {
+            source: ['https://wiki.smzdm.com/p/:id'],
```
Comment (TonyRL) [inline lib/routes/smzdm/product.ts:20]:
```suggestion
            source: ['wiki.smzdm.com/p/:id'],
```

File: lib/routes/smzdm/product.ts (hunk 2 above)
Comment (chesha1) [reply lib/routes/smzdm/product.ts:73]: On the RSSHub server side, what is the cache expiration period?

Also, what types of content should be cached? While browsing the source code of other routes, I noticed that some content is cached while other content is not. I'm wondering if there's a clear guideline for this.

**Pull request merged**